pkg/api/groups.go

```diff
@@ -559,8 +563,11 @@ func (api *API) GetGroupStatusCountTimeline(groupID string, duration string) (ma
 	// Get the versions and their number of instances per status within each of the given time intervals.
 	query := fmt.Sprintf(`
 	WITH time_series AS (SELECT * FROM generate_series(now() - interval '%[1]s', now(), INTERVAL '%[2]s') AS ts)
-	SELECT ts, (CASE WHEN status IS NULL THEN 0 ELSE status END), (CASE WHEN version IS NULL THEN '' ELSE version END), count(instance_id) as total FROM (SELECT * FROM time_series
-		LEFT JOIN LATERAL(SELECT * FROM instance_status_history WHERE group_id=$1 AND %[3]s AND created_ts BETWEEN time_series.ts - INTERVAL '1 hour' + INTERVAL '1 sec' AND time_series.ts) _ ON TRUE) AS _
+	SELECT ts, (CASE WHEN status IS NULL THEN 0 ELSE status END), (CASE WHEN version IS NULL THEN '' ELSE version END), count(instance_id) as total FROM (
+		SELECT * FROM time_series
+		LEFT JOIN (SELECT * FROM instance_status_history WHERE group_id=$1 AND %[3]s) _
+		ON created_ts BETWEEN time_series.ts - INTERVAL '1 hour' + INTERVAL '1 sec' AND time_series.ts
+		) AS _
```

Comment:
I'd decrease the indentation for this line:
```suggestion
	) AS _
```